python/cuml/cuml/manifold/umap.pyx

```diff
@@ -510,15 +520,10 @@ class UMAP(UniversalBase,
             else:
                 umap_params.build_algo = graph_build_algo.NN_DESCENT
                 build_kwds = self.build_kwds or {}
-                umap_params.nn_descent_params.graph_degree = <uint64_t> build_kwds.get("nnd_graph_degree", 64)
-                umap_params.nn_descent_params.intermediate_graph_degree = <uint64_t> build_kwds.get("nnd_intermediate_graph_degree", 128)
+                umap_params.nn_descent_params.intermediate_graph_degree = <uint64_t> build_kwds.get("nnd_intermediate_graph_degree", 0)
                 umap_params.nn_descent_params.max_iterations = <uint64_t> build_kwds.get("nnd_max_iterations", 20)
                 umap_params.nn_descent_params.termination_threshold = <float> build_kwds.get("nnd_termination_threshold", 0.0001)
-                umap_params.nn_descent_params.return_distances = <bool> build_kwds.get("nnd_return_distances", True)
```

Comment:
This option should never have been exposed, setting it to anything other than True would error.